config/projects/chefdk.rb

```diff
@@ -65,7 +65,12 @@
   override :rubygems,     version: "2.4.8"
 end
 
+# update to 0.9.1 when that is released
+# master currently has some dependency changes we need
+override :'inspec', version: "master"
+
 override :'test-kitchen', version: "v1.4.2"
+override :'kitchen-inspec', version: "master" # will set when release is tagged
```

Comment:
Just pushed kitchen-inspec to rubygems: https://rubygems.org/gems/kitchen-inspec